src/libraries/System.Text.Encoding/tests/Ascii/EqualsTests.cs

```diff
@@ -91,11 +114,14 @@ public static IEnumerable<object[]> EqualIgnoringCaseConsiderations
             {
                 yield return new object[] { "aBc", "AbC" };
 
-                for (char i = (char)0; i <= 127; i++)
+                foreach (int textLength in BufferLengths)
                 {
-                    char left = i;
-                    char right = char.IsAsciiLetterUpper(left) ? char.ToLower(left) : char.IsAsciiLetterLower(left) ? char.ToUpper(left) : left;
-                    yield return new object[] { new string(left, i), new string(right, i) };
+                    for (char chr = (char)0; chr <= 127; chr++)
```

Comment:
I see the value of this change, but we should decrease the number of test cases. How about just focusing on Ascii letters? Because for other characters we would be duplicating other tests work.